components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -65,8 +65,46 @@ const int kMaxReferralServerResponseSizeBytes = 1024 * 1024;
 // run.
 const char kDefaultPromoCode[] = "BRV001";
 
+namespace brave {
+
 namespace {
 
+BraveReferralsService::ReferralInitializedCallback
+    g_testing_referral_initialized_callback;
+
+base::FilePath g_promo_file_path;
+
+void DeletePromoCodeFile(const base::FilePath& promo_code_file) {
+  if (!base::DeleteFile(promo_code_file)) {
+    LOG(ERROR) << "Failed to delete referral promo code file "
+               << promo_code_file.value().c_str();
+    return;
```

Comment:
this isn't necessary

Reply:
yep, this was just copied and I didn't look closely enough at it